sdk/servicebus/pom.service.xml

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-servicebus-service</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
```

Comment:
Is that comment accurate?

Reply:
I don't know - it was there previously so I left it alone. We don't seem to publish this POM which is perhaps why this comment is there.

Reply:
Will follow up on this with the Service Bus team - leaving it alone in this PR.